client/a8c-for-agencies/sections/sites/controller.tsx

```diff
@@ -33,6 +35,7 @@ function configureSitesContext( isFavorites: boolean, context: Context ) {
 			filter={ filter }
 			sort={ sort }
 			showSitesDashboardV2={ true }
+			isFavoriteFilterInitialState={ isFavoriteFilter }
```

Comment:
Instead of creating a new property, we can use the existing `filter` one that already has the `showOnlyFavorites` filter field:

line 15:
```
const { s: search, page: contextPage, issue_types, sort_field, sort_direction, is_favorite } = context.query;
```

lines 16-19:
```
const filter = {
		issueTypes: issue_types?.split( ',' ),
		showOnlyFavorites: ( is_favorite === true || is_favorite === '' ),
	};
```

That means we don't need all the context code below because we already have it inside of the `filter` property.